api-client/src/utils/__tests__/GeoJson.spec.ts

```diff
@@ -0,0 +1,52 @@
+import LocationModel from '../../models/LocationModel'
+import { convertToPoint, embedInCollection } from '../GeoJson'
+import { Feature, FeatureCollection } from 'geojson'
+
+describe('GeoHelper Methods', () => {
+  const expectedGeoJsonMarkerFeature: Feature = {
+    type: 'Feature',
+    geometry: {
+      type: 'Point',
+      coordinates: [29.979848, 31.133859]
+    },
+    properties: {
+      name: 'Test'
+    }
+  }
+  describe('ConvertToPoint', () => {
```

Comment:
```suggestion
  describe('convertToPoint', () => {
```

Reply:
I'm still stuck in C# naming conventions 😂